client/src/helpers/nova/hooks/useAccountAddressState.ts

```diff
@@ -43,7 +48,8 @@ export const useAccountAddressState = (address: AccountAddress): [IAccountAddres
     );
 
     const { accountOutput, isLoading: isAccountDetailsLoading } = useAccountDetails(network, address.accountId);
-    const { totalBalance, availableBalance } = useAddressBalance(network, state.accountAddressDetails, accountOutput);
+    const { totalBalance, availableBalance } = useAddressBalance(network, state.addressDetails, accountOutput);
+    const [addressBasicOutputs, isBasicOutputsLoading] = useAddressBasicOutputs(network, state.addressDetails?.bech32 ?? null);
```

Comment:
What about other outputs ? Afaik all `CommontOutput` (SDK-nova) can have the `NativeTokenFeature`.

Reply:
On Nova only Basic and Foundry outputs can have NativeTokenFeature